cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature

```diff
@@ -124,3 +124,81 @@ Feature: SecureStorage_Manage
+            | removeForApp with device scope with 2 apps    | remove secure data value for an app with scope device   | set secure data value1 for an app with scope device  | set secure data value2 for an app with scope device  | get stored value for an app for authTestTokenDevice1 with scope device   | get stored value for an app for authTestTokenDevice2 with scope device   | expected value for authTestTokenDevice2 stored data in securestorage  | expected value for authTestTokenDevice1 stored data in securestorage  | clear stored value with scope as device for an app  |
+
+    @SecureStorage @manageSDK
+    Scenario Outline: SecureStorage.clearForApp - Positive Scenario: <Scenario>
+        Given 3rd party 'certification' app is launched with 'secondary 3rd party app' appId
+        Given 1st party app invokes the 'Firebolt' API to '<Set_API_Key1>'
+        And 'Firebolt' platform responds to '1st party app' for '<Set_API_Key1>'
+        And 1st party app invokes the 'Firebolt' API to 'set secure data value1 for second app with scope device'
+        And 'Firebolt' platform responds to '1st party app' for 'null for updating a secure data value for an app'
+        When '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        Then 'Firebolt' platform responds with '<Validation_Key1>'
+        When 'secondary 3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with '<Validation_Key1>'
+        When 1st party app invokes the 'Firebolt' API to '<Clear_API_Key>'
+        Then 'Firebolt' platform responds to '1st party app' for '<Clear_API_Key>'
+        When '3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        Then 'Firebolt' platform responds with 'null for getting stored value'
+        When 'secondary 3rd party app' invokes the 'Firebolt' API to '<Get_API_Key1>'
+        And 'Firebolt' platform responds to 'secondary 3rd party app' with '<Validation_Key1>'
+
+
+        Examples:
+            | Scenario                       | Clear_API_Key                                          | API_Key                                                      | Set_API_Key1                                         | Set_API_Key2                                         | Get_API_Key1                                                             | Get_API_Key2                                                             | Validation_Key2                                                       | Validation_Key1                                                       |
```

Comment:
can we add comments for each TC 
# Clearing data from app
# Getting value from app